spec/support/server.spec.ts

```diff
@@ -0,0 +1,39 @@
+import request from 'supertest';
+import server from '../../src/server';
+
+describe("server tests", () => {
+  it("responds to /", () => {
+    return request(server)
+      .get('/')
+      .set('Accept', 'application/json')
+      .expect(200);
+  });
+
+  it("responds to / with hello from clinical trial ", () => {
```

Comment:
We should probably remove the extra spaces from the end of the test descriptions.